WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -105,13 +105,15 @@ struct OrderPaymentSection: View {
             TitleAndValueRow(title: viewModel.couponSummary ?? Localization.coupon, value: .content(viewModel.discountTotal), selectionStyle: .highlight) {
                 shouldShowCouponLineDetails = true
             }
+            .disabled(viewModel.shouldDisableAddingCoupons)
```

Comment:
💯 

Not part of this PR, but we could possibly consider apply the same modifier to "+ Add Shipping" and "+ Add Fees".